_events/eventsRedirect.md

```diff
@@ -6,5 +6,7 @@ permalink: /events/
 
 ## SSE Events
 
+Hi there & welcome to our events page! SSE is proud to sponsor a number of events for MSOE SE and CE students. Check out our hack-n-tells, welcome week activities, company tours and mock-whiteboard interviews! From social networking to professional preparation, SSE hosts your destination events for professional, technical and social networking!
```

Comment:
Ooo, hack-n-tells like RokkinCat's? https://www.rokkincat.com/hack-n-tell/ 
If so, would it make sense to hyperlink that text to their hack-n-tell site?

Reply:
Exactly. We've had groups of SSE people attend their Hack-n-tells before, so linking to their site is a really great idea!

Reply:
I can add that link in there in this PR, or should we make a ticket to scan through the content and add appropriate links everywhere?

Reply:
Add the link here in the PR and for future content you can reference this work as an example. @GFLEMING133 thoughts?